homeassistant/components/sleepiq/strings.json

```diff
@@ -24,13 +24,16 @@
       }
     }
   },
-  "selector": {
-    "foot_warmer_temp": {
-      "options": {
-        "off": "Off",
-        "low": "Low",
-        "medium": "Medium",
-        "high": "High"
+  "entity": {
+    "select": {
+      "foot_warmer_temp": {
+        "name": "Foot Warmer Temperature",
```

Comment:
```suggestion
```

Reply:
Removed it since you're still including names and this PR needs entity translations as a whole to be implemented